code/modules/media/broadcast/transmitters/broadcast.dm

```diff
@@ -225,7 +225,7 @@
 			M.apply_radiation(round(rads*count_rad_wires()/2),RAD_EXTERNAL)
 
 		// Heat output
-		var/turf/simulated/L = loc
+		var/turf/simulated/L = get_turf(loc)
 		if(istype(L) && heating_power)
 			var/datum/gas_mixture/env = L.return_air()
```

Comment:
```suggestion
		var/datum/gas_mixture/env = loc?.return_air()
		if(istype(env) && heating_power)
```

Reply:
Actually this is not optimal because it should check `heating_power` before calling `return_air()` but that would require tabbing the stuff below by another tab which I don't want to do in a suggestion
You can accept this or write something yourself